src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -212,7 +213,7 @@ static void window_new_ride_scrollmouseover(rct_window *w, int32_t scrollIndex,
 static void window_new_ride_invalidate(rct_window *w);
 static void window_new_ride_paint(rct_window *w, rct_drawpixelinfo *dpi);
 static void window_new_ride_scrollpaint(rct_window *w, rct_drawpixelinfo *dpi, int32_t scrollIndex);
-static void window_new_ride_list_vehicles_for(const uint8_t rideType, const rct_ride_entry * rideEntry, char * out);
+static void window_new_ride_list_vehicles_for(uint8_t rideType, const rct_ride_entry* rideEntry, char* buffer, size_t bufferLen);
```

Comment:
Do we need to drop the `const` on `rideType`? Should we?

Reply:
We rarely make copy-by-value parameters immutable, it would add so much code bloat if we applied this to everything. In this case I just felt it was out of place, inconsistent with everything else here.